packages/amplify-util-mock/src/__tests__/utils/dynamo-db/util.test.ts

```diff
@@ -219,6 +219,7 @@ describe('DynamoDB Utils', () => {
         };
         if (typeof callback === 'function') {
           callback(null, response);
+          return undefined;
         } else {
```

Comment:
nit: Why don't we return nothing or `null` instead?

Reply:
`return undefined` preserves behavior.
It's not a goal of this PR to rework this test. Priority is to enable rules so new tests added are held to standard.